onnxruntime/core/providers/cpu/sequence/sequence_ops.h

```diff
@@ -53,4 +53,22 @@ class SequenceConstruct final : public OpKernel {
   }
   Status Compute(OpKernelContext* context) const override;
 };
+
+class SplitToSequence final : public OpKernel {
+ public:
+  SplitToSequence(const OpKernelInfo& info);
+  Status Compute(OpKernelContext* context) const override;
+
+ private:
+  template <typename T>
+  Status ComputeImpl(OpKernelContext& context, const Tensor& input, const Tensor* p_split_input) const;
+  Status PrepareForCompute(const TensorShape& input_shape, int64_t split_scalar, bool is_split_input_scalar,
+                           int64_t& num_outputs, int64_t& axis, int& before_dims,
+                           int& after_dims_including_split_axis, int& after_dims_excluding_split,
+                           bool& is_uneven_split, int& num_remaining_splits,
+                           std::vector<int64_t>& split_sizes) const;
+  int64_t axis_{};
```

Comment:
why is keepdims_ initialized here but not axis_?

Reply:
This is the C++ way of default initializing a variable to 0.